scripts/compile.sh

```diff
@@ -83,6 +83,7 @@ build_llvm() {
     cmake -G Ninja llvm -B build \
         -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS_DEBUG="-g -O0" \
         -DLLVM_ENABLE_ASSERTIONS=true -DLLVM_ENABLE_PROJECTS="mlir"\
+        -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD="SPIRV" \
```

Comment:
The llvm-version should be updated to bb4aeb66ab4f39901ef6c1d01baf8b82b81cecfb, otherwise the build fails with this option and the actual version from the main branch.